test/unit/events.test.js

```diff
@@ -239,3 +239,21 @@ test('should have relatedTarget correctly set on the event', function() {
 
   Events.trigger(el2, { type:'click', relatedTarget:undefined });
 });
+
+QUnit.test('should execute remaining handlers after an exception in an event handler', function(assert) {
+  assert.expect(1);
+
+  const el = document.createElement('div');
+  const listener1 = function() {
+    throw new Error('GURU MEDITATION ERROR');
+  };
+  const listener2 = function() {
+    assert.ok(true, 'Click Triggered');
```

Comment:
I am not a fan of assertions in callbacks since it breaks reading order but that's totally a nitpick and not worth holding up the PR for.